body.tex

```diff
@@ -18,71 +22,63 @@ \section{Introduction}
 
 In this document, we lay out the procedural details for how we define and carry out plans, following the framework described above, to deliver on our milestones, maintain visibility in our workflows, remain responsive to change, and offer staff the ability to innovate and collaborate.
 
-
-\section{Useful Contacts}
+\section{Rubin Program Management}
 \label{sec:contacts}
 
-The \RO Acting \gls{Director}  is Robert Blum, the Interim Deputy Director for NOIRLab is Amanda Bauer, and the Deputy Director for SLAC is Phil Marshall.
-% PJM: \RO macro needs a trailing \xspace
-They are the first point of contact for all issues regarding project management within \RO.
+The Rubin Observatory Acting \ \gls{Director}  is Robert Blum and the Interim Deputy \gls{Director} for NOIRLab is Amanda Bauer and the Deputy Director for SLAC is Phil Marshall.
 
+They are the first point of contact for all issues regarding project management within \RO.
 
-The  Project Controls Specialist is Cathy Petry.
-She monitors the budgets and maintains details within WeBud, the NOIRLab budgeting system. She assists in tracking milestones and reporting.
+The  Program Coordinator is Cathy Petry.
+She monitors the budgets and maintains details within WeBud, the NOIRLab budgeting system. She assists in developing the annual Program Operating Plan (POP), tracking milestones and reporting. On the SLAC side, Christine Soldhal, Business Manager, handles similar tasks. The POP is a defined report and process for NOIRLab. Rubin Operations considers the POP to be a Rubin activity, which is informs both NOIRLab and SLAC leadership of the annual Rubin activity including milestones and budget. For NOIRLab, the Rubin POP is integrated and delivered to NSF for the next fiscal year at the end of the current fiscal year. For SLAC, the POP informs SLACs annual planning which culminates in a Field Work Proposal (FWP) for all SLAC High Energy Physics activity including Rubin. The FWP is delivered in June of the current fiscal year for the next fiscal year. The FWP is previewed to SLAC management in February in advance of the final delivery in June. Because the POP for NSF and FWP for DOE are out of phase, Rubin does high level planning in early Q2 of the financial year. Detailed activity planning including lower level milestones continues though the year in advance of the next year. This detailed planning is the subject of this document.
```

Comment:
@drphilmarshall can you check this text please?

Reply:
```suggestion
She monitors the budgets and maintains details within WeBud, the NOIRLab budgeting system. She assists in developing the annual Program Operating Plan (POP), tracking milestones and reporting. On the SLAC side, Christine Soldahl, Business Manager, handles similar tasks. The POP is a defined report and process for NOIRLab. Rubin Operations considers the POP to be a Rubin activity, which is informs both NOIRLab and SLAC leadership of the annual Rubin activity including milestones and budget. For NOIRLab, the Rubin POP is integrated and delivered to NSF for the next fiscal year at the end of the current fiscal year. For SLAC, the POP informs SLAC's annual planning, which culminates in a Field Work Proposal (FWP) for all SLAC High Energy Physics activity including Rubin. The FWP is delivered in June of the current fiscal year, and covers the federal budget request for the next two fiscal year. The FWP is previewed to SLAC management and DOE in February in advance of the final delivery in June. Because the POP for NSF and FWP for DOE are out of phase, Rubin does high level planning in early Q2 of the financial year. Detailed activity planning including lower level milestones continues though the year in advance of the next year. This detailed planning is the subject of this document.
```